fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/postgresql/PostgreSqlTranslator.java

```diff
@@ -113,7 +113,7 @@ public boolean isDuplicateSchema(SQLException x) {
 
     @Override
     public boolean isUndefinedName(SQLException x) {
-        return "42X05".equals(x.getSQLState());
```

Comment:
Maybe this should be an OR with 

```
42883 | undefined_function
42P01 | undefined_table
42P02 | undefined_parameter
42704 | undefined_object
```

Reply:
empirically I found the `42704` is what we got back for undefined functions, tables, and indexes.  not sure how to reconcile that with the provided list, but its easy-enough for me to or them

Reply:
I wasn't sure if we should or not, it was more of a check-in, not necessarily that we had to or-then

Reply:
ok, i went ahead and added them